src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+unreleased
++++++++++++++++++++
+* api version 2017-07-01 support
+* update dcos and swarm to use latest api version instead of 2016-03-30
+* major refactoring
+
```

Comment:
'Major refactoring' isn't useful for the end-user.
If you can expand on that on any changes that affect your commands (what an end-user will see), please include that; otherwise I'd remove this line.